tests/functional/test_tracing.py

```diff
@@ -13,6 +14,38 @@
 mountpoint = "/mnt"
 
 
+def test_iotracer_multiple_instances():
+    """
+        title: Check if it’s possible to run concurrent tracing processes.
+        description: |
+          Try to run multiple concurrent io-tracers for the same disks.
+        pass_criteria:
+          - No system crash.
+          - It’s impossible to run io-tracer multiple times for the same disks.
+    """
+    with TestRun.step("Run the first io-tracer instance"):
+        iotracer1 = IotracePlugin(TestRun.plugins['iotrace'].repo_dir,
+                                  TestRun.plugins['iotrace'].working_dir)
+        iotracer1.start_tracing()
+
+    with TestRun.step("Check if the first instance of io-tracer works."):
+        if not iotracer1.check_if_tracing_active():
+            TestRun.LOGGER.error("There is no working io-tracer instance.")
+        TestRun.LOGGER.info(f"Iotrace process found with PID {iotracer1.pid}")
+
+    with TestRun.step("Try to start the second iotracer instance."):
+        iotracer2 = IotracePlugin(TestRun.plugins['iotrace'].repo_dir,
+                                  TestRun.plugins['iotrace'].working_dir)
+        iotracer2.start_tracing()
+        if iotracer2.check_if_tracing_active():
```

Comment:
This could potentially have a race condition, since tracing is started in the background.
I'd use a sleep for one second in between.